notebooks/HASP/FluxScaleTutorial/FluxScaleTutorial.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Please remember to use a context manager whenever possible. For examples, refer to the feedback I left on the Coadd notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>